### ISSUE TITLE
Border on Checkbox is not visible ,on office ui fabric in high contrast White mode unlike High Contrast Black mode and normal mode

### ISSUE THREAD

**Issue by v-ajrai** [microsoft/fluentui#16228]
### Environment Information
OS: Win 10
Browser:  Edge

### Describe the issue:

Border on Checkbox is not visible ,on office ui fabric in high contrast White mode unlike High Contrast Black mode and normal mode
**repro link:**
https://developer.microsoft.com/en-us/fluentui#/controls/web/checkbox
images
**_HC_white_image_with_issue_img:_**
![HC_white_checbox_external_issue](https://user-images.githubusercontent.com/62233982/102195561-be3bfc80-3ee4-11eb-8db7-321e69b5ec6b.png)
**_HC_black_image_no_issue_img:_**
![HC_black_checbox_external_issue](https://user-images.githubusercontent.com/62233982/102195566-c005c000-3ee4-11eb-80c7-7bda4d76416e.png)
**_Normal_mode_image_no_issue_img:_**
![Normal_checbox_external_issue](https://user-images.githubusercontent.com/62233982/102195606-cdbb4580-3ee4-11eb-848b-2c493ca7a2cf.png)

#### Actual behavior:

Border on Checkbox is not visible ,on office ui fabric in high contrast White mode unlike High Contrast Black mode and normal mode

#### Expected behavior:

Border on Checkboxshould be visible ,on office ui fabric in high contrast White mode like High Contrast Black mode and normal mode

**Comment by ecraig12345** [comment]
Checkbox uses `ActiveBorder` for the focus border color, but for some reason that doesn't seem to be implemented properly in the high contrast white theme?? I also noticed that Checkbox is the only component which uses that color, so I think we should just switch it to use `WindowText` for the focus border instead.
